gtk4/Gir.toml

```diff
@@ -2138,10 +2150,24 @@ manual_traits = ["WidgetExtManual"]
         const = true
     [[object.function]]
     name = "get_name"
-    rename = "get_widget_name"
+    rename = "widget_name"
     [[object.function]]
     name = "set_name"
     rename = "set_widget_name"
+    [[object.function]]
+    name = "get_sensitive"
+    # Avoid clash with existing `is_sensitive` with a different meaning.
```

Comment:
What's the difference meaning of is_sensitive/is_visible here?

Reply:
See https://developer.gnome.org/gtk3/stable/GtkWidget.html#gtk-widget-get-sensitive and `gtk_widget_is_sensitive` below. I'd be glade if someone could find names that make the difference obvious :)

Reply:
I think we should rather rename is_sensitive to `is_effective_sensitive` (not great i know, but any suggestions are more than welcome) as per the getter rename rules it's get_sensitive that should become is_sensitive. The is_sensitive case is similar as well. As both those functions check whether the current widget is sensitive/visible along with it's parents as well.

Reply:
I didn't use this because it would change the behaviour of existing code. Maybe we should keep `is_sensitive` / `is_visible` as is, meaning that we test whether the widget is `sensitive` / `visible` in the resulting UI, and `is_widget_...` for the value of the widget itself... or we keep `get_...` :)

Reply:
On a second thought, `is_widget_...` isn't helpful... I guess that's why I kept `get_...`. Finally `get_property_...` would be the most explicit I guess.

Reply:
So, I kept `get_{sensitive,visible}` and improved the comments in `Gir.toml`, so that we don't need to do all the thinking again :)

Reply:
Thanks @fengalin

Reply:
Your're welcome!